Jenkinsfile

```diff
@@ -3,6 +3,17 @@ pipeline {
         label 'jenkins-workers'
     }
 
+    options {
+        timestamps()
+        buildDiscarder(logRotator(numToKeepStr: "10")) // keep only last 10 builds
+    }
+    
+    environment {
+        BUILD_TAG = sh label: 'Generating build tag', returnStdout: true, script: 'python3 pipeline/scripts/tag.py ${GIT_BRANCH} ${BUILD_NUMBER} ${GIT_COMMIT}'
+        BUILD_TAG_LOWER = sh label: 'Lowercase build tag', returnStdout: true, script: "echo -n ${BUILD_TAG} | tr '[:upper:]' '[:lower:]'"
+        ENVIRONMENT_ID = "build"
```

Comment:
I would give the environment a different name - there could be a conflict between the current MHS build environment